dnf/cli/cli.py

```diff
@@ -858,10 +858,10 @@ def configure(self, args, option_parser=None):
         # with cachedir in place we can configure stuff depending on it:
         self.base._activate_persistor()
 
-        self._configure_repos(opts)
-
         self.base.configure_plugins()
 
+        self._configure_repos(opts)
```

Comment:
After caused that plugin debuginfo-install cannot enable debug-info repository therefore the upgrade of debuginfo package is not working (plugin-path-3.feature)